tests_system/lobster_pkg/test_valid_scenario.py

```diff
@@ -0,0 +1,121 @@
+from pathlib import Path
+from .lobster_pkg_system_test_case_base import LobsterPKGSystemTestCaseBase
+from ..asserter import Asserter
+
+import os
+import shutil
+from tempfile import TemporaryDirectory
+
+
+class InputFilePkgTest(LobsterPKGSystemTestCaseBase):
+
+    def setUp(self):
+        super().setUp()
+        self._test_runner = self.create_test_runner()
+        self.temp_dir = TemporaryDirectory()  # pylint: disable=R1732
+
+    def _replace_working_directory(self, temp_out_file):
+        # Get the current working directory
+        current_working_dir = os.getcwd()
+
+        # Replace WORKING_DIRECTORY in the temp output file
+        with open(temp_out_file, 'r') as f:
+            content = f.read()
+
+        # Replace the placeholder with actual working directory (with forward slashes)
+        modified_content = content.replace('WORKING_DIRECTORY',
+                                            current_working_dir.replace('\\', '/'))
+
+        with open(temp_out_file, 'w') as f:
+            f.write(modified_content)
+
+    def test_valid_input_pkg_file(self):
+        OUT_FILE = "sample1.lobster"
+        self._test_runner.cmd_args.files = [str(
+            self._data_directory / "sample1.pkg")]
```

Comment:
Unusual formatting. Please update as follows:
```suggestion
        self._test_runner.cmd_args.files = [
            str(self._data_directory / "sample1.pkg"),
        ]
```